src/library/autodj/dlgautodj.ui

```diff
@@ -138,6 +138,26 @@ If no track sources are configured, the track is added from the library instead.
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QLabel" name="fadeModeLabel">
+       <property name="text">
+        <string>Fade mode:</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QComboBox" name="fadeModeCombobox"/>
+     </item>
+     <item>
+       <widget class="QCheckBox" name="repeatPlaylistCheckbox"/>
+     </item>
+     <item>
+      <widget class="QLabel" name="repeatPlaylistLabel">
+       <property name="text">
+        <string>Repeat playlist</string>
```

Comment:
How about replacing it with a repeat button with an icon?

Reply:
Can we just remove this change here? It is unrelated to the transition and there were no user requests about this.

Reply:
I have been wanting to make this change for a long time. Also, we occasionally get users asking how to enable this feature. They do not discover that it exists because they do not think to look in the preferences.

Reply:
There is simply no space. Even the mode combobox is already  a problem if you considder translations. Let's delay it until we have a three column layout. In 2.4. there is plenty of space for things like that.

Reply:
How about we replace all the text on AutoDJ buttons with icons? Then there will be plenty of space. @ronso0 do you have ideas for icons?

Reply:
you refer to existing buttons?
I don't see a problem using standard icons:
Enable > **Power** icon
Fade Now > **FFW/Next** icon
Skip Track > **Skip** icon https://amcollect.com/wp-content/uploads/2017/08/skip-tracing-icon-1.png
Shuffle > **shuffle** icon
Add random > **Dice** icon?

Reply:
Please let's postpone this and keep this PR focussed.
Some time ago we have rejected a PR introducing Auto DJ icons. 
I really want to have this PR merged and advance to 2.3 beta.

Reply:
Okay. Would you prefer I revert the commit or rebase to remove it?

Reply:
I don't mind ... reabase probably.

Reply:
There is no need anymore now that the "Fade mode" label is gone:
![image](https://user-images.githubusercontent.com/9455094/64084087-24e7d780-cced-11e9-890c-0fe664e53f0e.png)

Reply:
The tree is still unusable in my case. This is that worse, because it is independent of auto DJ table visible or not. Also note, that English is one if the shortest languages.
I guess French people's will have already issues with only the combo box added.

So please remove the "Repeat playlist"  checkbox as a quick fix.
Alternative: a burger menu ...

Reply:
Why is the tree not usable for you? I would rather change the checkbox to a button with an icon than get rid of it. For me it is important to know the state of this option. I have accidentally had repeated music when I did not intend it because I forgot this option was on.

Reply:
This now works again after the recent changes, Thank you.